registration/include/pcl/registration/impl/correspondence_rejection_sample_consensus.hpp

```diff
@@ -110,8 +110,7 @@ CorrespondenceRejectorSampleConsensus<PointT>::getRemainingCorrespondences(
       return;
     }
 
-    pcl::Indices inliers;
-    sac.getInliers(inliers);
+    const auto inliers = sac.getInliers();
```

Comment:
```suggestion
    const auto& inliers = sac.getInliers();
```
Otherwise it will do a copy again. `auto` does not mean that `inliers` will be a reference, the reference has to be explicitly requested. (a few similar cases below)